### ISSUE TITLE
unimport doesn't remove previous imports when there is an assignment with the imported module's name 

### ISSUE THREAD

**Issue by isik-kaplan** [hakancelikdev/unimport#120]
For the snippet below, unimport doesn't remove the first import, but it is not used.
```py
import datetime
datetime = None
import datetime
```

I think it might be that, unimport thinks the assignment below is a usage for the `datetime` variable, while it is not. 

Running unimport on the code produces
```
←[32m✨ Congratulations there is no unused import in your project. ✨←[0m
```

**Comment by isik-kaplan** [comment]
Just a heads up, if you commit with github's syntax, AKA `Fixes #120` github automatically closes the issue. Also, thanks!

**Comment by hakancelik96** [comment]
The PR has not closed yet, so please don't close the issue.
